### DISCUSSION TITLE
CNCF TAG-Runtime Discussion/Presentation (?)

### DISCUSSION THREAD

**Discussion by raravena80** [unikraft/unikraft#511] [General]
Hello Unikraft team,

I'm one of the co-chairs of the CNCF TAG-Runtime. I believe you had an agenda item to present the project in our meeting. I'm following up, I think it would be great for you to present/discuss the project in the future.

Feel free to add it to our [agenda](https://bit.ly/cncf-tag-runtime-meeting-notes) or reach out to me (raravena80 at gmail.com) at your convenience. 

Thanks!

**Comment by nderjung** [comment]
Hi @raravena80,

I've added Unikraft as a pending agenda item again.  I'll confer with the core team to let you know who will be able to join the next TAG-Runtime meeting. Thanks!

_Edit: @razvand and I will attend on 1st sept._
